Documentation/Resources.md

```diff
@@ -35,6 +35,10 @@ Also, include the verbose build log. If you're using `swift build` to compile th
 
     $ swift build -v
 
+If you are finding interesting behavior when updating your package, you can get verbose information via:
+
+    $ swift package --verbose update
```

Comment:
We already have verbose explained above, so I don't think this is needed.

Reply:
I was having a tough time finding it (via `git grep` and github search) when looking for package resolution information- would this be better as

```
add `--verbose` or `-v` after the subcommand to obtain the verbose log, such as:

    $ swift build -v
```

If not, no worries and feel free to close! 🙇

Reply:
Yeah, that makes sense! Can you update the PR?

Reply:
Thanks! Tried to incorporate it a bit better